package.json

```diff
@@ -2,6 +2,11 @@
   "scripts": {
     "test": "xo"
   },
+  "dependencies": {
+    "grunt": "^1.0.1",
+    "grunt-cli": "^1.2.0",
+    "grunt-webstore-upload": "^0.8.10"
+  },
```

Comment:
@paulmolluzzo First thing - having `npm install --dev` in Travis CI is completely fine. You don't install your test dependencies in production, but you do install them on Travis CI to run them. Second, it should be in the `install` stage, not `before_install`. `before_install` is for things you need to perform the actual install, e.g download/install/update the dependency manager if you don't have it provided by Travis etc.

I'm fine with installing just on Travis and not requiring for development, if others think it's cleaner this way, but I think we should keep them in `devDependencies`.